src/TestSuite/IntegrationTestTrait.php

```diff
@@ -752,7 +752,8 @@ protected function _url($url)
     {
         $uri = new Uri($url);
         $path = $uri->getPath();
-        $query = $uri->getQuery();
+        parse_str($uri->getQuery(), $query);
+        $query = http_build_query($query, null, '&', PHP_QUERY_RFC3986);
```

Comment:
Why are we parsing and then re-encoding the query string now?

Reply:
It's due to difference in how query string params are encoded in values generated by `http_build_query()` and value returned by `Uri::getQuery()`. I have explained it my comment https://github.com/cakephp/cakephp/pull/13955#issuecomment-559854498.